test/src/tests/generated/httpinfrastructuregroup/models.go

```diff
@@ -21,8 +21,9 @@ type A struct {
 
 // B ...
 type B struct {
-	TextStatusCode *string `json:"textStatusCode,omitempty"`
-	StatusCode     *string `json:"statusCode,omitempty"`
+	autorest.Response `json:"-"`
```

Comment:
This is the return type of `Created` I mentioned in the comment above.
Despite no functions are returning this type, we still add a `autorest.Response` to it...